common/metrics/queries.go

```diff
@@ -65,7 +65,7 @@ const (
 		"filter (resource.instance_id == '%s') | group_by 1m, [value_utilization_mean: mean(value.utilization)] " +
 		"| every 1m | group_by [], [value_utilization_mean_aggregate: aggregate(value_utilization_mean)]"
 	spannerStorageUtilDbQuery = "fetch spanner_instance | metric 'spanner.googleapis.com/instance/storage/used_bytes' | " +
-		"filter (resource.instance_id == '%s') && (metric.database == '%s') | " +
+		"filter (metric.database == '%s') && (resource.instance_id == '%s') | " +
```

Comment:
This this swap intentional ?

Reply:
Yes, remember I mentioned there a bug, the issue was that I put the instance name in db name and vice versa 😅

Reply:
Also for error handline, that is part of this PR: https://github.com/GoogleCloudPlatform/spanner-migration-tool/pull/686
I have tested failure scenarios as mentioned in the description